### PR TITLE
Remove regional price buttons

### PR DESCRIPTION
**Pull request by Revadike**
Resolves https://github.com/tfedor/AugmentedSteam/issues/474

image

### CODE REVIEW

File: js/options/options.js [hunk 1]
```diff
@@ -387,6 +387,7 @@ let Options = (function(){
             return `<div class="country_parent">
             <span class='es_flag ${countryClass}'></span>
             <select class='regional_country'>${options}</select>
+            <button type="button" name="profile_custom_remove" class="custom-link__close"></button>
```
Comment (MxtOUT) [inline js/options/options.js]: Why did you give it that `name`?

File: js/options/options.js [hunk 2]
```diff
@@ -395,12 +396,22 @@ let Options = (function(){
             let countries = SyncedStorage.get("regional_countries");
             countries.forEach(country => {
                 HTML.beforeBegin(addAnotherWrapper, generateRegionSelect(country));
+                addAnotherWrapper.previousSibling.querySelector(".custom-link__close").addEventListener("click", e => {
+                    let select = e.target.closest(".country_parent").querySelector(".regional_country");
+                    select.value = "";
+                    saveOption("regional_countries");
```
Comment (MxtOUT) [inline js/options/options.js]: This is the same listener as 10 lines below

File: js/options/options.js (hunk 1 above)
Comment (Revadike) [reply js/options/options.js]: I think I copied it. I'll check and see if i can remove/change it.

Comment (Revadike) [reply js/options/options.js]: Removed. That shouldn't affect anything.

File: js/options/options.js (hunk 2 above)
Comment (Revadike) [reply js/options/options.js]: Abstracted it

**Pull request merged**